freeipa/src/main/java/com/sequenceiq/freeipa/service/TlsSecurityService.java

```diff
@@ -103,9 +103,8 @@ private void generateSaltPassword(SaltSecurityConfig saltSecurityConfig) {
         saltSecurityConfig.setSaltPasswordVault(saltPassword);
     }
 
-    public GatewayConfig buildGatewayConfig(Long stackId, InstanceMetaData gatewayInstance, Integer gatewayPort,
+    public GatewayConfig buildGatewayConfig(Stack stack, InstanceMetaData gatewayInstance, Integer gatewayPort,
```

Comment:
if we get stack here instead of id we can omit the gw port from parameters as it's in the stack already

Reply:
fixed